spec/set_order_command_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'set_order_command'
+
+RSpec.describe SetOrderCommand do
```

Comment:
These are great tests that really focus on the behaviour of `SetOrderCommand`! :+1:

Small repetition I noticed could be reduced with using RSpec's `let` helpers, i.e.
```
let(:event_data) {{user_id: "asdf", user_name: "a name" }}
```
at the beginning of the `describe` block.

https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/helper-methods/let-and-let